scripts/create-user.js

```diff
@@ -0,0 +1,228 @@
+  message: 'Please add your description: (optional)',
+  validate: validateDescription
+};
+
+function validateCountry(value) {
+  const pass = value.match(
+    /^[A-Za-z].{3,}$/
+  )
+  if (pass) {
+    return true;
+  }
+  return 'Please enter a valid country. Minimum 4 characters.';
+}
+const questionCountry = {
+  type: 'input',
+  name: 'country',
+  message: 'Please add your country:',
+  validate: validateCountry
+};
+
+function validateTags(value) {
+  if (value) {
+    const array = value.split(',');
+    if (array.length > 0 && array.length < 6) {
+      return true;
+    }
+  }
+  return 'Please enter valid tags. Between 1 and 5 tags.';
+}
+const questionTags = {
+  type: 'input',
+  name: 'tags',
+  message: 'Please add your tags: (Separate by commas)',
+  validate: validateTags
+};
+
+function validateChannels(answer) {
+  if (answer.length < 1) {
+    return 'You must choose at least one option.';
+  } else if (answer.length > 3) {
+    return 'You must choose maximum three options'
+  }
+  return true;
+}
+const questionChannels = {
+  type: 'checkbox',
+  name: 'channels',
+  message: 'Please add your channels:',
+  choices: [
+    {
+      name: 'Email'
+    },
+    {
+      name: 'Slack'
+    },
+    {
+      name: 'Linkedin'
+    },
+    {
+      name: 'Facebook'
+    },
+    {
+      name: 'Twitter'
+    }
+  ],
+  validate: validateChannels
+};
+
+const questionByChannel = {
+  email: {
+    type: 'input',
+    name: 'Email',
+    message: 'Please add your contact email:',
+    validate: validateEmail,
+    when: (answers) => answers.channels.find((channel) => channel === 'Email')
+  },
+  slack: {
+    type: 'input',
+    name: 'Slack',
+    message: 'Please add your Slack member id:',
+    when: (answers) => answers.channels.find((channel) => channel === 'Slack')
+  },
+  linkedin: {
+    type: 'input',
+    name: 'Linkedin',
+    message: 'Please add your Linkedin user id:',
+    when: (answers) => answers.channels.find((channel) => channel === 'Linkedin')
+  },
+  facebook: {
+    type: 'input',
+    name: 'Facebook',
+    message: 'Please add your Facebook user id:',
+    when: (answers) => answers.channels.find((channel) => channel === 'Facebook')
+  },
+  twitter: {
+    type: 'input',
+    name: 'Twitter',
+    message: 'Please add your Twitter user id:',
+    when: (answers) => answers.channels.find((channel) => channel === 'Twitter')
+  }
+}
+
+const questions = [
+  questionEmail,
+  questionName,
+  questionAvatar,
+  questionTitle,
+  questionDescription,
+  questionCountry,
+  questionTags,
+  questionChannels,
+  questionByChannel.email,
+  questionByChannel.slack,
+  questionByChannel.linkedin,
+  questionByChannel.facebook,
+  questionByChannel.twitter
+];
+
+async function convertAnswersToSchema(answers) {
+  const choices = ['Email', 'Slack', 'Linkedin', 'Facebook', 'Twitter'];
+  for (let answer in answers) {
+    if (answer === 'tags') {
+      answers[answer] = answers[answer].split(',');
```

Comment:
I would also call [`trim`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim) to avoid tags with whitespace.

```suggestion
      answers[answer] = answers[answer].split(',').map(t => t.trim());
```